### PR TITLE
[READY_TO_REVIEW]feature/#435 Anchors redirectioning to collections in table relations

### LINKED ISSUE #435: Collection "anchors"

### PR DESCRIPTION
**Pull request by Ivanruii**
closed #435

### CODE REVIEW

File: src/pods/canvas/components/canvas-accessible/components/table-relations-accessible.component.tsx [hunk 1]
```diff
@@ -22,6 +25,7 @@ export const TableRelationsAccessible: React.FC<Props> = props => {
             <TableRelationElementOrigin
               relation={relation}
               originTable={table}
+              originTableRef={collectionRefs.current[table.id]}
```
Comment (deletidev) [inline src/pods/canvas/components/canvas-accessible/components/table-relations-accessible.component.tsx:28]: destinarionTableRef={collectionRefs.current[relation.toTableId]}

File: src/pods/canvas/components/canvas-accessible/components/table-relations-accessible.component.tsx [hunk 2]
```diff
@@ -32,6 +36,7 @@ export const TableRelationsAccessible: React.FC<Props> = props => {
             <TableRelationElementDestination
               relation={relation}
               destinationTable={table}
+              destinationTableRef={collectionRefs.current[relation.fromTableId]}
```
Comment (deletidev) [inline src/pods/canvas/components/canvas-accessible/components/table-relations-accessible.component.tsx:39]: originTableRef={collectionRefs.current[relation.fromTableId]}

File: src/pods/canvas/components/canvas-accessible/components/table-relations-accessible-element-origin.component.tsx [hunk 3]
```diff
@@ -27,11 +28,25 @@ export const TableRelationElementOrigin: React.FC<Props> = props => {
     relation.toFieldId
   );
 
+  const scrollToTablePosition = () => {
```
Comment (deletidev) [inline src/pods/canvas/components/canvas-accessible/components/table-relations-accessible-element-origin.component.tsx:31]: Rename, with something like, focusTablePosition

File: src/pods/canvas/components/canvas-accessible/components/table-relations-accessible-element-destination.component.tsx [hunk 4]
```diff
@@ -32,6 +34,13 @@ export const TableRelationElementDestination: React.FC<Props> = props => {
     relation.toFieldId
   );
 
+  const scrollToTablePosition = () => {
```
Comment (deletidev) [inline src/pods/canvas/components/canvas-accessible/components/table-relations-accessible-element-destination.component.tsx:37]: Rename, with something like, focusTablePosition

File: src/pods/canvas/components/canvas-accessible/components/table-relations-accessible-element-destination.component.tsx [hunk 5]
```diff
@@ -10,11 +10,13 @@ import { findFieldNameAndParent } from './table-relations-accessible.business';
 interface Props {
   relation: RelationVm;
   destinationTable: TableVm;
+  destinationTableRef: React.RefObject<HTMLHeadingElement>;
```
Comment (deletidev) [inline src/pods/canvas/components/canvas-accessible/components/table-relations-accessible-element-destination.component.tsx:13]: Rename with originTableRef

File: src/pods/canvas/components/canvas-accessible/components/table-relations-accessible-element-origin.component.tsx [hunk 6]
```diff
@@ -5,11 +5,12 @@ import { findFieldNameAndParent } from './table-relations-accessible.business';
 interface Props {
   relation: RelationVm;
   originTable: TableVm;
+  originTableRef: React.RefObject<HTMLHeadingElement>;
```
Comment (deletidev) [inline src/pods/canvas/components/canvas-accessible/components/table-relations-accessible-element-origin.component.tsx:8]: Rename with destinationTableRef

File: src/pods/canvas/components/canvas-accessible/components/table-relations-accessible.component.tsx (hunk 1 above)
Comment (Ivanruii) [reply src/pods/canvas/components/canvas-accessible/components/table-relations-accessible.component.tsx:28]: Are you sure I need to change the prop name? I see it as the table in which is the origin of the relationship.

Wouldn't that be something like this?:
`originTableRef={collectionRefs.current[relation.toTableId]}`

**Pull request merged**